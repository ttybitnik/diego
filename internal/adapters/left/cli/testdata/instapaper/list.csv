URL,Title,Selection,Folder,Timestamp
https://www.newyorker.com/magazine/2017/09/04/fernando-pessoas-disappearing-act,Fernando Pessoaâ€™s Disappearing Act,,Starred,1679174385
https://dougseven.com/2014/04/17/knightmare-a-devops-cautionary-tale/,Knightmare: A DevOps Cautionary Tale,,Starred,1694613937
https://www.newyorker.com/magazine/2016/08/08/lauren-collins-learns-to-love-in-french,Love in Translation,,Starred,1679174483
