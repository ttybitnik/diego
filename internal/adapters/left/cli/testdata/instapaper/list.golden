- name: Fernando Pessoaâ€™s Disappearing Act
  url: https://www.newyorker.com/magazine/2017/09/04/fernando-pessoas-disappearing-act
  timestamp: "1679174385"
- name: 'Knightmare: A DevOps Cautionary Tale'
  url: https://dougseven.com/2014/04/17/knightmare-a-devops-cautionary-tale/
  timestamp: "1694613937"
- name: Love in Translation
  url: https://www.newyorker.com/magazine/2016/08/08/lauren-collins-learns-to-love-in-french
  timestamp: "1679174483"
